test/quantization/test_quantization.py

```diff
@@ -1363,6 +1363,24 @@ def checkModelWithSequentialQuantized(self, model):
         self.assertEqual(type(model.classifier[0]), nniq.LinearReLU)
         self.assertEqual(type(model.classifier[1]), nn.Identity)
 
+    def test_fusion_conv_with_bias(self):
+        model = ModelForFusionWithBias().train()
+        model = fuse_modules(model, [["conv1", "bn1", "relu1"],
+                                     ["conv2", "bn2"]])
+        model.qconfig = default_qat_qconfig
```

Comment:
This code only changes prepare_qat, so no need to test after convert. We should call prepare_qat with identity fake_quant, so that the numerics of the model are identical to the original model. 
```
model.qconfig = QConfig(activation=torch.nn.Identity,
                                      weight=torch.nn.Identity)
prep_model = prepare_qat(model, inplace=False)